grafana-plugin/src/containers/IntegrationContainers/ExpandedIntegrationRouteDisplay/ExpandedIntegrationRouteDisplay.tsx

```diff
@@ -195,28 +199,34 @@ const ExpandedIntegrationRouteDisplay: React.FC<ExpandedIntegrationRouteDisplayP
                       Escalation chain
                     </InlineLabel>
                     <WithPermissionControlTooltip userAction={UserActions.IntegrationsWrite}>
-                      <GSelect
-                        showSearch
+                      <Select
+                        isSearchable
                         width={'auto'}
-                        modelName="escalationChainStore"
+                        menuShouldPortal
                         className={cx('select', 'control')}
                         placeholder="Select escalation chain"
                         isLoading={isRefreshingEscalationChains}
-                        displayField="name"
                         onChange={onEscalationChainChange}
-                        value={channelFilter.escalation_chain}
+                        options={Object.keys(escalationChainStore.items).map(
+                          (eschalationChainId: EscalationChain['id']) => ({
+                            id: escalationChainStore.items[eschalationChainId].id,
+                            value: escalationChainStore.items[eschalationChainId].name,
```

Comment:
Some weirdness here as it relies on value apparently. For now I'll let both value and label to be the same, it needs `value` for searching